main/src/ca/uwaterloo/flix/api/lsp/LocationLink.scala

```diff
@@ -85,6 +85,17 @@ object LocationLink {
     LocationLink(originSelectionRange, targetUri, targetRange, targetSelectionRange)
   }
 
+  /**
+    * Returns a reference to the instance node `node`.
```

Comment:
Returns a location link to the given `instance`.

Reply:
Ok!